docs/examples/State Estimation Example.ipynb

```diff
@@ -36,10 +36,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 4,
```

Comment:
please clear output and re-run all cells from scratch. this also provides a double-check that whatever you implemented is indeed correct